Sprint_Tests/maria_ptitsyna.md

```diff
@@ -0,0 +1,12 @@
+1. He used to manage well in layout and work with actual technologies in front end. After small break in several years and major depression plus listening hard music, he weakly manages in actual tendentious and drinks a lot. If he had overcome the depression after dead of his wife and child, he wouldn't have had a question about search of a job with a relevant salary.
+
+2. If he had a goal and comparable wage expectation, he would already work remotely somewhere in the UAS or in Bali with a perspective to relocate in The Canary Islands and make spanish citizens. Since childhood, he has been dreaming about marriage with spanish girl and got retired early and growing up grape on a seaside of The Balearic Sea.
+
+3. If you don't know how a piece of code, which was written by a colleague, works, you have you cover it by console.log and trace all logic chain. This is a time-consuming action. If we had documentation to this feature or ticket for feature realization with description of business logic, we would have flowed out the fix during the couple of hours.
+
+4. I want you to feel free to express your thoughts in foreign language in spite of
+mistakes, which can appear because of absence of practice and infrequency of language classes. I want every single one of you to read one book in the original before the end of the year, language of original is naturally English, and get pleasure from overcome a language speak barrier. 
```

Comment:
in a foreign language
the mistakes
the absence
the language of the origin/original 
from overcoming (Gerund after prepositions)
language barrier